tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+envlist = flake8,pydocstyle,docs
+
+
+[testenv]
+basepython=python2.7
+# Installing with sdist makes tox/pip very unhappy.
+usedevelop=True
+# Bodhi requires koji, which isn't available on PyPI.
+sitepackages=True
+
+
+[testenv:docs]
+changedir = docs
+deps =
+    sphinx
+    cornice_sphinx
+whitelist_externals =
+    mkdir
+    rm
+    sphinx-build
+commands=
+    rm -rf _build images
+    mkdir -p _static images
+    ./generate_db_schema
+    sphinx-build -W -b html -d {envtmpdir}/doctrees .  _build/html
+    sphinx-build -W -b man -d {envtmpdir}/doctrees .  _build/man
+
+
+[testenv:flake8]
+deps =
+    flake8 > 3.0
+commands =
+    python -m flake8 {posargs}
+
+
+[testenv:pydocstyle]
+deps =
+    pydocstyle
+whitelist_externals =
+    pydocstyle
+commands =
+    pydocstyle
+
+
+[pydocstyle]
+inherit = false
+ignore = D413
+match =
+    bodhi/server/views/*.py
+    bodhi/server/consumers/__init__.py
+    bodhi/server/consumers/signed.py
+    bodhi/server/consumers/updates.py
+    bodhi/server/scripts/
+    bodhi/server/services/stacks.py
+    bodhi/server/views/
+
+
+[flake8]
+show-source = True
+max-line-length = 100
+exclude = .git,.tox,dist,*egg
+ignore = E712
+
+
+[pytest]
+addopts = --junit-xml=nosetests.xml --cov-config .coveragerc --cov=bodhi --cov-report term --cov-report xml --cov-report html
```

Comment:
You could rename ```nosetests.xml``` to whatever you like (maybe just ```test_results.xml```), but it'll require coordination of the JJB service job (which only runs when things are merged to develop) and changing this. Thus, I recommend making a separate PR after this one is merged that does that and only that, because any PR that makes that change is guaranteed to fail CI. We can just make an exception. We can also just leave it named this way forever too.